packages/application/src/shell.ts

```diff
@@ -308,6 +308,7 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
     const skipLinkWidget = (this._skipLinkWidget = new Private.SkipLinkWidget(
       this
     ));
+    this.add(skipLinkWidget, 'top', { rank: 0 });
```

Comment:
Oki I look a bit into this. And we are hit by a CSS rule preventing the skip link to show.

In `packages/application/style/skiplink.css` could you add `contain: none !important;` to the `.jp-skiplink-wrapper` rule? (and remove the `add` call)

Then the focus is not on the skip link because the focus is on the main area element (I need to press like 8 time Shift-Tab to get on the Skip link widget using the code on the `main` branch). Do you know how we could improve that?

Reply:
resolved by  commit 383dfdd